packages/normalizr/src/entities/Entity.ts

```diff
@@ -219,6 +219,20 @@ First three members: ${JSON.stringify(input.slice(0, 3), null, 2)}`;
           addEntity,
           visitedEntities,
         );
+      } else if (process.env.NODE_ENV !== 'production') {
+        const error = new Error(
```

Comment:
same question as on the other PR; throwing an error (as opposed to just logging) in only dev seems like it would not be desired, since that would mean prod and dev actually do different things.

Reply:
Given the other PR, how do you feel about landing this for now but keeping conversation open?